spec/requests/users_spec.rb

```diff
@@ -14,24 +14,23 @@
 
     it 'should include correct placeholder' do
       get '/users'
-      expect(response.body).to include('<h1>This is a list of users</h1>')
+      expect(response.body).to include('<h1 class="heading">RailsBlog</h1>')
```

Comment:
- Please test the requested features. Below are the things to test for;

`User Index`
- Tests that asserts you can see the username of all other users.
- Tests that asserts you can see the profile picture for each user.
- Tests that asserts you can see the number of posts each user has written.
- Tests that redirects to the user's show page when you click on a user.

`User show`
- Tests that asserts you can see the user's profile picture.
- Tests that asserts you can see the user's username.
- Tests that asserts you can see the number of posts the user has written.
- Tests that asserts you can see the user's bio.
- Tests that asserts you can see the user's first 3 posts.
- Tests that asserts you can see a button that lets me view all of a user's posts.
- Tests that redirects to the post's show page when you click on a user's post.
- Tests that redirects to the user's post's index page when you click on a **see all posts**.